app/_src/gateway-operator/support.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Support
+badge: enterprise
+content-type: reference
+---
+
+Learn about the support for {{site.kgo_product_name}} software versions.
+
+## Version support for {{site.kgo_product_name}}
+
+Kong primarily follows [semantic versioning](https://semver.org/) (SemVer) for its products.
+
+At Kong’s discretion a specific minor version can be marked as a LTS version. The LTS version is supported on a given distribution for the duration of the distribution’s lifecycle, or for 3 years from LTS release whichever comes sooner. LTS only receives security fixes or certain critical patches at the discretion of Kong. Kong guarantees that at any given time, there will be at least 1 active LTS Kong version.
+
+LTS versions of {{site.kgo_product_name}} are supported for 3 years after release. Standard versions are supported for 1 year after release.
```

Comment:
What is our definition of a standard version? It's just a major version in semver terms right? So `v1.x.x` and we'll support that for 1 year with minor and patch releases?